test-network-function/generic/suite.go

```diff
@@ -302,23 +318,23 @@ func testIsRedHatRelease(oc *interactive.Oc) {
 // TODO: Multus is not applicable to every CNF, so in some regards it is CNF-specific.  On the other hand, it is likely
 // a useful test across most CNFs.  Should "multus" be considered generic, cnf_specific, or somewhere in between.
 var _ = ginkgo.Describe(multusTestsKey, func() {
-	if testcases.IsInFocus(ginkgoconfig.GinkgoConfig.FocusStrings, multusTestsKey) {
-		config := GetTestConfiguration()
-		log.Infof("Test Configuration: %s", config)
+	//	if testcases.IsInFocus(ginkgoconfig.GinkgoConfig.FocusStrings, multusTestsKey) {
```

Comment:
Should this (multus focus) be part of this commit?

Reply:
no. thanks for spotting it, I'll fix it